### PR TITLE
Fixed init command when category in yml is wrong

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36304

## Description
Fixed init command when category in yml is wrong

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/init/initiator.py [hunk 1]
```diff
@@ -98,7 +98,8 @@ class Initiator:
                 INDICATOR_TYPES_DIR, REPORTS_DIR, WIDGETS_DIR, DOC_FILES_DIR]
 
     def __init__(self, output: str, name: str = '', id: str = '', integration: bool = False, template: str = '',
-                 script: bool = False, pack: bool = False, demisto_mock: bool = False, common_server: bool = False):
+                 category: str = '', script: bool = False, pack: bool = False, demisto_mock: bool = False,
```
Comment (roysagi) [inline demisto_sdk/commands/init/initiator.py:101]: why add a category here?

File: demisto_sdk/commands/init/initiator.py [hunk 2]
```diff
@@ -232,6 +234,7 @@ def pack_init(self) -> bool:
             fill_manually = user_response in ['y', 'yes']
 
             pack_metadata = Initiator.create_metadata(fill_manually)
+            self.category = pack_metadata['categories'][0]
```
Comment (roysagi) [inline demisto_sdk/commands/init/initiator.py:237]: not sure if pack categories are the same as integration categories - need to check that - you can look at the validations to make sure.

File: demisto_sdk/commands/init/initiator.py [hunk 3]
```diff
@@ -240,7 +243,7 @@ def pack_init(self) -> bool:
         if create_integration in ['y', 'yes']:
             integration_init = Initiator(output=os.path.join(self.full_output_path, 'Integrations'),
                                          integration=True, common_server=self.common_server,
-                                         demisto_mock=self.demisto_mock, template=self.template)
+                                         demisto_mock=self.demisto_mock, template=self.template, category=self.category)
```
Comment (roysagi) [inline demisto_sdk/commands/init/initiator.py:246]: might be smarter to add a question to the user (using `input`) of which integration category does he want to add

**Review by roysagi — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/init/initiator.py (hunk 3 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/init/initiator.py]: done.

**Review by roysagi — APPROVED** [review]
Looks good and good demo!
Please just fix the changelog entry

**Pull request merged**